deployment/network-operator/values.yaml

```diff
@@ -106,6 +106,8 @@ operator:
   # tag, if defined will use the given image tag, else Chart.AppVersion will be used
   # tag
 
+imagePullSecrets: []
```

Comment:
we need to support image pull secrets in sriov network operator as well

Reply:
This can be achieved (if imagePullSecrets cannot be easily set in sriov-network-operator and its config daemon)
by adding imagePullSecrets to service account:

https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account

Reply:
NFD already implemented this https://github.com/kubernetes-sigs/node-feature-discovery/blob/566957624036d172a0d39081d10193cc09cfdeb2/deployment/helm/node-feature-discovery/values.yaml#L7